cinema/urls.py

```diff
@@ -0,0 +1,10 @@
+from django.urls import path
+
+from cinema.views import movie_list, movie_detail
+
+urlpatterns = [
+    path("movies/", movie_list, name="movie-list"),
+    path("movies/<int:pk>", movie_detail, name="movie-detail"),
```

Comment:
According to the checklist, all your endpoints should end with `/`. Please add a trailing slash to your `movie-detail` path. It should look like this: `movies/<int:pk>/`.